polaris-react/src/components/Tooltip/components/TooltipOverlay/TooltipOverlay.scss

```diff
@@ -18,23 +18,21 @@
   transform: none;
   transition: none;
 
-  #{$se23} & {
-    @include shadow-bevel(
-      $boxShadow: var(--p-shadow-md),
-      $borderRadius: var(--pc-tooltip-border-radius),
-      $zIndex: var(--pc-tooltip-shadow-bevel-z-index),
-      // The following arguments explicitly ignore the shadow-bevel opt out
-      $border: none,
-      $content: ''
-    );
-  }
+  @include shadow-bevel(
+    $boxShadow: var(--p-shadow-md),
+    $borderRadius: var(--pc-tooltip-border-radius),
+    $zIndex: var(--pc-tooltip-shadow-bevel-z-index),
+    // The following arguments explicitly ignore the shadow-bevel opt out
```

Comment:
I wonder if we can delete these now 🤔 I added it as the last point to [this epic](https://github.com/Shopify/polaris/issues/9915) so we can check across all the mixin uses at the end!